src/app/modules/interface-status-icon/interface-status-icon.component.html

```diff
@@ -1,11 +1,7 @@
-@if (isLinkUp()) {
-  <ix-icon
-    [id]="elementId"
-    [name]="statusIcon()"
-    [matTooltip]="tooltipText()"
-  ></ix-icon>
-} @else {
-  <ix-icon name="ix-network-upload-download-disabled"></ix-icon>
-}
-
+<ix-icon
+  [id]="elementId"
+  [name]="statusIcon()"
+  [matTooltip]="isLinkUp() ? tooltipText() : null"
```

Comment:
As a user, I would expect to see a tooltip with a status when the network interface is down

Reply:
Thanks for a review @denysbutenko . I'll add a tooltip. Good point.

Reply:
@denysbutenko - that's it:
<img width="656" alt="Screenshot 2025-04-12 at 14 07 12" src="https://github.com/user-attachments/assets/68f4ccb1-d50d-4bd1-b52e-d998c2d006ef" />